pkg/runtime/initError.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+import (
+	"fmt"
+)
+
+type InitErrorKind string
+
+const (
+	InitFailure            InitErrorKind = "InitFailure"
```

Comment:
If we want these to be printed out, we should agree if these should be app caps, camel case or pascal case. Last but not least, we should agree on these error types since these become contractual.

I personally think these should be ALL_CAPS and we can keep these failure types here.

/cc @yaron2

Reply:
I agree on ALL_CAPS. cc @daixiang0

Reply:
I agree on ALL_CAPS as well.